internal/controller/imagepolicy_controller_test.go

```diff
@@ -177,12 +177,11 @@ func TestImagePolicyReconciler_ignoresImageRepoNotReadyEvent(t *testing.T) {
 		return err == nil && conditions.IsReady(imagePolicy)
 	}).Should(BeTrue())
 
-	// Wait a bit and check that the ImagePolicy remains ready.
-	time.Sleep(time.Second)
```

Comment:
Good catch adding `, timeout, interval` below! However, the reason why this sleep is here is to ensure that even after some time the ImagePolicy remains ready, and we didn't just "get lucky" in the assertion right above (please add `, timeout, interval` there as well :pray:)

Reply:
Thanks for the catch! 

The `time.Sleep` is there to ensure the ImagePolicy remains ready even after some time, not just immediately after the ImageRepository
  becomes stalled. I've:
  - Added the missing `timeout, interval` to the first assertion
  - Kept the `time.Sleep` to test the stability over time
  - Added `timeout, interval` to the second assertion